source/load-balancers/tests/lb-connection-establishment.yml

```diff
@@ -0,0 +1,28 @@
+description: connection establishment for load-balanced clusters
+
+schemaVersion: '1.3'
+
+runOnRequirements:
+  - loadBalanced: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      uriOptions:
+        # Explicitly set loadBalanced to false to override the option from the global URI.
+        loadBalanced: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+
+tests:
+  - description: operations against load balancers fail if URI contains loadBalanced=false
```

Comment:
This test should fail. It expects the server to fail the connection handshake because it will not contain `loadBalanced: true`, but support for this has not been implemented in the server yet.